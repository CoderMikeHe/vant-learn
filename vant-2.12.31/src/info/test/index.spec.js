import Info from '..';
import { mount } from '../../../test';

test('should not render when badge is empty string', () => {
  const wrapper = mount(Info, {
    propsData: {
      badge: '',
    },
  });

  expect(wrapper).toMatchSnapshot();
});

test('should not render when badge is empty undefined', () => {
  const wrapper = mount(Info, {
    propsData: {
      badge: undefined,
    },
  });

  expect(wrapper).toMatchSnapshot();
});

test('should render when badge is zero', () => {
  const wrapper = mount(Info, {
    propsData: {
      badge: 0,
    },
  });

  expect(wrapper).toMatchSnapshot();
});
