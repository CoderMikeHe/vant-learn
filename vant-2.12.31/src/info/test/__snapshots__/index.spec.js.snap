// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should not render when badge is empty string 1`] = ``;

exports[`should not render when badge is empty undefined 1`] = ``;

exports[`should render when badge is zero 1`] = ``;
